public/index.html

```diff
@@ -137,6 +137,37 @@ <h2 class="text-xl md:text-2xl md:text-4xl font-extrabold">Ticket Sales Start In
                 </article>
             </section>
             <hr class="text-slate-800">
+            <div class="max-w-4xl mx-auto p-6">
+                <h1 class="text-2xl md:text-4xl font-extrabold text-center">Keynote Speakers</h1>
+                <div class="grid grid-cols-1 md:grid-cols-3 gap-6 mt-8">
+                  <div class="relative bg-slate-800 border-slate-700 border rounded-xl p-4 shadow-lg text-center">
+                    <img src="images/speakers/irina_nazarova_1.png" alt="Irina Nazarova" class="w-48 h-48 mx-auto rounded-full mb-4">
+                    <h3 class="text-xl font-semibold text-red-600 my-3">Irina Nazarova</h3>
+                    <p class="text-slate-100 text-sm">CEO at Evil Martians, co-founder at AnyCable <br> Organizer of the SF Ruby meetup</p>
+                    <a class="absolute inset-0" target="_blank" href="https://www.linkedin.com/in/nonconstant/"></a>
+                  </div>
+                  <div class="relative bg-slate-800 border-slate-700 border rounded-xl p-4 shadow-lg text-center">
+                    <img src="images/speakers/ridhwana_khan_1.jpeg" alt="Speaker 2" class="w-48 h-48 mx-auto rounded-full mb-4">
+                    <h3 class="text-xl font-semibold text-red-600 my-3">Ridhwana Khan</h3>
+                    <p class="text-slate-100 text-sm">Technical Writer at The Rails Foundation <br> Community Builder</p>
```

Comment:
I'm not sure we should skip the "Lead Engineer at Circle" part, I'd rather not have the "Community Builder" if we need it to be short.

Reply:
Okay!

TBH I was not sure what to put exactly :)

Reply:
Here is the preview link:
https://4d1e1cd7.rubyconfth.pages.dev/

The thing is that maybe we should put lead eng first and then the rails foundation?

I'm not sure, also maybe we should make them all shorter?